### PR TITLE
Pages for Fast Choose Wheel

### PR DESCRIPTION
**Pull request by feytox**
## Added
- Pages for Fast Choose Wheel.
- Page select buttons in ModernChooseWheel.

## Changed
- Config for "fastmenu" to new for pages. Old config automatically converted to new version.

## Demonstration
pages_for_fast_choose_wheel

### CODE REVIEW

**Comment by DearFox** [comment]
The old version of the wheel does not have toggles and will always display the last emote page selected (0 by default). It's banal there is no convenient place to add it. For the most part, this is what I redesigned the emotion selection wheel for.

**Comment by DearFox** [comment]
The arrows for switching pages do not have a selection visualization. The reason is the lack of space on the texture for these purposes.

File: emotesMain/src/main/java/io/github/kosmx/emotes/main/config/ClientConfig.java [hunk 1]
```diff
@@ -64,7 +64,8 @@ public ClientConfig(){
     //public final EmoteHolder[] fastMenuEmotes = new EmoteHolder[8];
 
     public BiMap<UUID, InputKey> emoteKeyMap = new BiMap<>();
-    public UUID[] fastMenuEmotes = new UUID[8];
+    public ConfigEntry<Integer> fastMenuPage = new IntegerConfigEntry("fastMenuPage", 0, false, expert);
```
Comment (KosmX) [inline emotesMain/src/main/java/io/github/kosmx/emotes/main/config/ClientConfig.java]: I don't think you should use a config entry to store the last used page.
I think a static variable in the ModernChooseWheel is better.

**Review by KosmX — CHANGES_REQUESTED** [review]

**Review by KosmX — APPROVED** [review]
This looks good!
Thank you

**Pull request merged**